src/components/ProjectSettingsTabs/GithubCommitQueueTab/getFormSchema.ts

```diff
@@ -71,6 +74,13 @@ export const getFormSchema = (
                 aliasArray.schema
               ),
             },
+            githubTriggerAliases: {
+              type: "array" as "array",
+              title: "GiHub Trigger Aliases",
```

Comment:
Can we maybe add some help text explaining that they can be added on the Patch Aliases page?